controller/conf/application.conf

```diff
@@ -742,6 +742,9 @@ play.cache {
   #bindCaches = ["db-cache", "user-cache", "session-cache"]
 }
 
+#Keycloak optional config
```

Comment:
Remove Keycloak word from this and other comments and varibles. We can mention it as Auth Enabled

Reply:
resolved